examples/chart/teleport-cluster/values.yaml

```diff
@@ -516,13 +517,28 @@ rbac:
   # Set to false if your cluster level resources are managed separately.
   create: true
 
-# Options for the Teleport service
+# Options for the Teleport proxy service
+# This setting only applies to the proxy service. The teleport auth service is internal-only and always uses a ClusterIP.
+# You can override the proxy's backend service to any service type (other than "LoadBalancer") here if really needed.
+# To use an Ingress, set service.type=ClusterIP and ingress.enabled=true
 service:
   type: LoadBalancer
   # Additional entries here will be added to the service spec.
   spec: {}
     # loadBalancerIP: "1.2.3.4"
 
+# Options for ingress
```

Comment:
We should document that L7 Lbs requires using tsh as an intermediate proxy in order to have successful connections - people don't read docs :cry: